src/components/common/PageLayout/SideDrawer.tsx

```diff
@@ -17,7 +17,7 @@ type SideDrawerProps = {
 }
 
 const isSafeAppRoute = (pathname: string, query: ParsedUrlQuery): boolean => {
-  return pathname === AppRoutes.apps && !!query.appUrl
+  return pathname === AppRoutes.apps.index && !!query.appUrl
```

Comment:
now when the function only checks for the index route, should it be renamed too?

Reply:
I think that the name should be the same because this function checks if you are in the Safe App page, and the url is always like: 
```
.../apps?appUrl=https%3A%2F%2Fswap.cow.fi
```

Reply:
But now there are new subroutes:
```
    bookmarked: '/apps/bookmarked',
    custom: '/apps/custom',
```

They are technically Safe App routes, aren't they? But the function would return false for them. Maybe `isSafeAppFrameRoute` is a better name

Reply:
done!